src/components/Menu.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <oc-application-menu name="coreMenu" v-model="sidebarIsVisible" @close="sidebarIsVisible = false">
-    <oc-sidebar-nav-item v-for="(n, nid) in nav" :key="nid" :icon="n.iconMaterial" :target="n.route.path">{{ n.name }}</oc-sidebar-nav-item>
+    <oc-sidebar-nav-item v-for="(n, nid) in nav" :key="nid" :icon="n.iconMaterial" :target="n.route.path">{{ $gettext(n.name) }}</oc-sidebar-nav-item>
```

Comment:
:nauseated_face:

Reply:
If we need to use it then still in the component, wouldn't it be better with computed property and using $gettext directly there?

Reply:
yes - but this is anyhow wrong .....